content/docs/04-clusters/01-public-cloud/01-aws.md

```diff
@@ -1018,19 +1018,37 @@ sigs.k8s.io/cluster-api-provider-aws/cluster/[ClusterName] = owned
 ## Running Clusters through Edit Node Pool
 <InfoBox>

```

Comment:
This does not need to be in an info box, it should be the heading's introductory paragraph.